src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
+
+  --link-color: #00acdc;
+  --header-height: 60px;
+
+  font-family: Roboto, Arial, sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+  display: inline-block;
+}
+
+.imgblock {
+  height: 40px;
+  width: 40px;
+}
+
+header {
+  font-family: Roboto, sans-serif;
+  height: var(--header-height);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.navListItem {
+  font-style: normal;
+  align-content: center;
+  margin-left: 20px;
+}
```

Comment:
You should be consistent with your vertical margins. In this case, you've added a left margin. If you need to add a vertical margin, add only a top or bottom margin, but not both. This is to prevent margin collapsing.